source/renderer/app/components/profile/analytics/CollectedDataOverview.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { injectIntl } from 'react-intl';
+import { CollapsibleSection } from '../../widgets/collapsible-section/CollapsibleSection';
+import styles from './CollectedDataOverview.scss';
+import { messages } from './CollectedDataOverview.messages';
+
+export const CollectedDataOverview: FC = injectIntl(({ intl }) => {
```

Comment:
Do we have access to `useIntl` in this version?

Reply:
Unfortunately not. We use older version that does not provide hooks.